keylime/registrar_client.py

```diff
@@ -1,15 +1,41 @@
 import logging
+import ssl
 import sys
+from typing import Any, Dict, Optional, Union
 
 from keylime import api_version as keylime_api_version
 from keylime import crypto, json, keylime_logging
 from keylime.requests_client import RequestsClient
 
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+if sys.version_info >= (3, 11):
+    from typing import NotRequired
+else:
+    from typing_extensions import NotRequired
```

Comment:
@kkaarreell : This one here now creates an issue on the test machines.

Reply:
I have this here in requirements. Is there anything else that the test suite needs?

```
typing-extensions>=4.0.1 ; python_version < '3.11' #PSF-2.0
```

Reply:
Packit e2e tests are not using these dependencies but rather install them in 
https://github.com/RedHat-SP-Security/keylime-tests/blob/main/setup/install_upstream_keylime/test.sh#L40
as we prefer distribution packages. 
I need to check what do we have in current Fedora releases before fixing it.

Reply:
It looks like Fedora 37 would have the right version of the package (v4.2.0) but on Fedora 36 it's only v3.10: https://pkgs.org/download/python3-typing-extensions

Reply:
I did required changes in https://github.com/RedHat-SP-Security/keylime-tests/pull/301 but tests are failing on Rawhide due to missing keylime_ima_emulator service. I need to investigate it, it's weird.

Reply:
Seems that the new systemd (intentionally?) changed the output of `systemctl is-enabled`. I have a patch available, it is being crunched in CI and I will merge it once it passes.

Reply:
Done, please try to rerun tests by adding a comment "/packit test"

Reply:
failure on testing-farm:fedora-36-x86_64  is something I am investigating right now. You can try to rerun it once or twice and it shoud pass (/packit retest-failed) or eventually ignore.

Reply:
This time it passed.